src/components/dashboard/main-content/course-enrollments/course-cards/ContinueLearningButton.jsx

```diff
@@ -38,32 +40,39 @@ const ContinueLearningButton = ({
 
   const isCourseStarted = () => dayjs(startDate) <= dayjs();
   const isExecutiveEducation2UCourse = EXECUTIVE_EDUCATION_COURSE_MODES.includes(mode);
-  const execClassName = (isExecutiveEducation2UCourse) && (!isCourseStarted()) ? ' disabled btn-outline-secondary' : undefined;
+  const disabled = !isCourseStarted() ? 'disabled' : undefined;
+  const variant = isExecutiveEducation2UCourse ? 'inverse-primary' : 'primary';
 
   const renderContent = () => {
-    if (isExecutiveEducation2UCourse && !isCourseStarted() && startDate) {
-      const formattedStartDate = dayjs(startDate).format('MMM D, YYYY');
-      return `Available on ${formattedStartDate}`;
+    // resumeCourseRunUrl indicates that learner has made progress, available only if the learner has started learning.
```

Comment:
@adamstankiewicz I understood from reviewing the docs and comments that the `resumeCourseRunUrl` is returned if the learner has made progress. 
However, I'm not entirely certain if this is still the case with the enterprise-course-enrollments API endpoint...

That said, if the resumeCourseRunUrl isn't returned by the API endpoint anymore, do you have any idea how we can determine if the learner has made progress? Or would it be necessary to include that field in order to handle this scenario?

For testing purposes, I've mocked the data in the enterprise-course-enrollments API response, including the resumeCourseRunUrl field and I was able to get the cards display "Resume" button.

<img width="1066" alt="Screenshot 2023-11-15 at 11 45 22 AM" src="https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/113524403/1476eb2d-f50c-4d8e-8a41-e441f1704463">
<img width="749" alt="Screenshot 2023-11-15 at 11 48 54 AM" src="https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/113524403/5f1ec666-0adc-4817-af34-113c2422d685">
<img width="508" alt="Screenshot 2023-11-15 at 11 53 54 AM" src="https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/113524403/710d4833-ad1b-4563-b0a8-3cbd6a93933f">

Reply:
@mahamakifdar19 

> I understood from reviewing the docs and comments that the resumeCourseRunUrl is returned if the learner has made progress. However, I'm not entirely certain if this is still the case with the enterprise-course-enrollments API endpoint...

Yeah, I'm also a bit confused why some of the code in this app assumes the `resumeCourseRunUrl` is there, and documents it. I can't seem to find any history of that field existing in the `enterprise-course-enrollments` API response either... very strange. According to Git blame, it appears `resumeCourseRunUrl` has been present within the MFE since around February 14, 2020. Checking out an older commit `edx-enterprise` from around the same time as it was introduced in this MFE, I can't seem to find any notion of `resume_course_run_url` being returned by `enterprise-course-enrollments` either.

> That said, if the resumeCourseRunUrl isn't returned by the API endpoint anymore, do you have any idea how we can determine if the learner has made progress? One approach I'm considering is to include a condition where if the course has started and the courseRunUrl exists, we'll view that as indicating progress in learning. Your thoughts?

Without any additional data provided by the API, I think your suggestion that's the best we could do given the data the app has to work with, though it's not quite semantically correct (course may be started, without the learning having taken action yet).

Searching for `resume_course_run_url` in edx-platform, I came across how resume course URLs are generated for B2C learners (e.g., on home.edx.org) [here](https://github.com/openedx/edx-platform/blob/94658f58905abee5adf3fc8a899e9d8ea9a5a4b8/lms/djangoapps/learner_home/views.py#L261-L280) with `get_resume_urls_for_course_enrollments`, where given a user and a list of course enrollments, it returns a list of `resume_course_urls`.

If we need to differentiate between "course has started" and "learner has started", perhaps the `enterprise-course-enrollments` API could somehow implement/re-use a similar function such that `resumeCourseRunUrl` on enterprise enrollments is no longer `undefined`?

Reply:
@adamstankiewicz  I agree with you! 
For now, I have removed the usage of `resumeCourseRunUrl` and reverted things back to their previous state with the current available data. If the course has started, we display the resume button just as it was before. 

However, we do need to differentiate between 'course has started' and 'learner has started' but implementing a logic that returns 'resume_course_run_url' from the enterprise-course-enrollments API seems time consuming and many tickets are already dependent on this one. Perhaps we can create a separate ticket in the next sprint to address this change. What are your thoughts?

Reply:
> Perhaps we can create a separate ticket in the next sprint to address this change. What are your thoughts?

Completely agree! 😄 Making support for `resume_course_run_url` is definitely its own chunk of work and can be deferred from this PR/ticket. 

[inform] Just dropped a ✅ on the PR :shipit: